kubetest/dind/dind.go

```diff
@@ -348,6 +357,47 @@ ApiserverLoop:
 	}
 	log.Printf("apiserver is now available")
 
+NodeHealthLoop:
+	// Wait for the expected number of nodes to be ready.
+	for {
+		log.Printf("Waiting for nodes....")
+		select {
+		case <-d.control.Interrupt.C:
+			return fmt.Errorf("timed out waiting for nodes to be ready from cluster container %s", d.containerID)
+		case <-pollCh:
+			nodes, err := d.apiserver.CoreV1().Nodes().List(metav1.ListOptions{})
+			if err != nil {
+				log.Printf("No response for nodes....")
+				continue
+			}
+			// TODO(Q-Lee): take this from a flag.
+			numNodes := 4
```

Comment:
I think kubetest may have a flag for this now... too many flags :/

Reply:
extend from [--gcpNodes](https://github.com/leblancd/test-infra/blob/cc5d69e9a24a3a03a6bd4ccc9bcdf66518043db2/kubetest/main.go#L157)?

Reply:
we should migrating gcpNodes to a numNodes flag :-)